server/chat-commands/room-settings.ts

```diff
@@ -7,6 +7,7 @@
  * @license MIT
  */
 import {Utils} from '../../lib/utils';
+import type {RoomPermission} from '../user-groups';
```

Comment:
Moment of truth: how many copies of PS crash with this code in?

This requires a relatively more recent version of sucrase than before, and not everyone remembers that compile errors mean you should rerun `npm install`.

Reply:
Normally we use feature-detection and auto-reinstall, but I'm not sure if there's an easy way to do that for sucrase.

Reply:
```
> require('sucrase').getVersion()
'3.15.0'
```

AHA

Can you add this check to the reinstall checker?

Reply:
like https://github.com/smogon/pokemon-showdown/blob/master/build#L66 ?

Reply:
Actually, I will.

Reply:
Yeah, don't worry about that.